### DISCUSSION TITLE
Is it possible to change the behavior of middle clicking on a tab to open a sibling tab?

### DISCUSSION THREAD

**Discussion by zeroarst** [piroor/treestyletab#3262] [Q&A]
As tittle...
Current middle click close the tab. But I want to change it to open a new sibling tab, or even better like other options in TST.
I know I can do that by middle click bottom plus button. But it is not intuitive.

**Comment by piroor** [comment]
TST has no built-in option about that. Instead TST provides API to help implementing such a feature:
https://github.com/piroor/treestyletab/wiki/API-for-other-addons#when-a-tab-is-clicked

Indeed there are some addons providing extra behaviors for click actions on TST tabs:
https://addons.mozilla.org/firefox/addon/tst-middleclick/
https://addons.mozilla.org/firefox/addon/tst-change-close-tab-button-be/

Thus I recommend you to develop a new helper addon providing the feature you want and publish that - it will help people who have similar demand.
